setup.sh

```diff
@@ -1,22 +1,25 @@
 #!/usr/bin/env bash
 
-# If an error occurs, stop this script
-set -e
+# use bash strict mode
+set -euo pipefail
+IFS=$'\n\t'
+
+# uncomment for debug
+#set -x
 
 printf "== Installing local Ruby gems ==\n"
+gem install bundler
 bundle install --quiet --path=gems --binstubs
 
 #printf "\n== Uninstalling Ruby gems no longer used ==\n"
 bundle clean
 
 printf "\n== Installing node packages (used by gulp and iso via Node) ==\n"
-npm install npm@10.8.1
+npm install npm@10.2.4
```

Comment:
Why is this going back to an older version of npm?

Reply:
Good catch, it should not, I will fix it